plugins/text-freetype2/text-freetype2.c

```diff
@@ -468,6 +476,8 @@ static void *ft2_source_create(obs_data_t *settings, obs_source_t *source)
 	obs_data_set_default_int(font_obj, "size", 32);
 	obs_data_set_default_obj(settings, "font", font_obj);
 
+	obs_data_set_default_int(font_obj, "log_lines", 6);
```

Comment:
This sets a default value for the `font_obj` obs_data_t object.  This caused you not to actually set any default value because you were setting it on the wrong object.  You're supposed to use `settings` here instead.